### PR TITLE
Add Japanese translation installation.mdx

### LINKED ISSUE #18413: Tranformers documentation translation to Japanese 🇯🇵
Hi!

Let's bring the documentation to all the Japanese-speaking community :)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

- Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
- Please translate in a gender-neutral way.
- Add your translations to the folder called `ja` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
- Register your translation in `ja/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
- Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @omarespejel and @sgugger for review.
- 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Get Started section
- [x] [index.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.mdx) | https://github.com/huggingface/transformers/pull/21186
- [ ] [quicktour.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.mdx).
- [x] [installation.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.mdx) | https://github.com/huggingface/transformers/pull/21241

## Tutorial section
- [ ] [pipeline_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.mdx) 
- [ ] [autoclass_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/autoclass_tutorial.mdx) 
- [ ] [preprocessing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.mdx) 
- [ ] [training.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.mdx) 
- [ ] [accelerate.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.mdx) 
- [ ] [model_sharing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.mdx) 
- [x] [multilingual.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.mdx) | https://github.com/huggingface/transformers/pull/21084

### PR DESCRIPTION
**Pull request by kambehmw**
# What does this PR do?

Adds Japanese translation to installation.mdx

Partially addresses #18413 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@omarespejel @sgugger

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
LGTM, thanks! can you check too @younesbelkada

File: docs/source/ja/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+## ソースからのインストール
+
+以下のコマンドでソースから🤗 Transformersをインストールします:
+
+```bash
+pip install git+https://github.com/huggingface/transformers
+```
+
+このコマンドは最新の安定版ではなく、開発における最新のmainバージョンをインストールします。mainバージョンは最新の開発状況に対応するのに便利です。例えば、最後の公式リリース以降にバグが修正されたが、新しいリリースがまだ展開されていない場合などです。しかし、これはmainバージョンが常に安定しているとは限らないことを意味します。私たちはmainバージョンの運用を維持するよう努め、ほとんどの問題は通常、数時間から1日以内に解決されます。もし問題に遭遇した場合は、より早く修正できるように[Issue](https://github.com/huggingface/transformers/issues)を作成してください！
```
Comment (younesbelkada) [inline docs/source/ja/installation.mdx:92]:
```suggestion
このコマンドは最新の安定版ではなく、開発における最新の`main`バージョンをインストールします。`main`バージョンは最新の開発状況に対応するのに便利です。例えば、最後の公式リリース以降にバグが修正されたが、新しいリリースがまだ展開されていない場合などです。しかし、これは`main`バージョンが常に安定しているとは限らないことを意味します。私たちは`main`バージョンの運用を維持するよう努め、ほとんどの問題は通常、数時間から1日以内に解決されます。もし問題に遭遇した場合は、より早く修正できるように[Issue](https://github.com/huggingface/transformers/issues)を作成してください！
```

**Review by younesbelkada — APPROVED** [review]
Thanks a lot for the translation! Left a small nit, otherwise LGTM

File: docs/source/ja/installation.mdx (hunk 1 above)
Comment (kambehmw) [reply docs/source/ja/installation.mdx]: @younesbelkada 
Thanks for the review!
I have modified the documentation as you suggested.

Comment (younesbelkada) [reply docs/source/ja/installation.mdx]: Thank you so much @kambehmw !

**Pull request merged**